### DISCUSSION TITLE
How to resolve stack variable overlaps

### DISCUSSION THREAD

**Discussion by zcutlip** [NationalSecurityAgency/ghidra#5149] [Q&A]
Sometimes I retype stack variables lots of code disappears from decompilation, presumably due to variable overlaps that couldn't be reconciled. I get the following warning:

```
/* WARNING: Could not reconcile some variable overlaps */
```

This happens particularly in really large functions with lots of in-line calls within, where the callee's stack variables overlap those of the caller. 

Is there a way to identify which variables are causing problems so I can manually fix/retype them? Are there other manual steps I can take to clean things up (and undead my dead code)?

**Comment by jrmuizel** [comment]
@caheckman anything you can share about this?
